cirq-core/cirq/devices/thermal_noise_model_test.py

```diff
@@ -342,3 +342,19 @@ def test_noisy_moment_two_qubit():
             [9.87330937e-01, 0, 0, 9.95013725e-01],
         ],
     )
+
+
+def test_thermal_noise_model_repr_and_json():
```

Comment:
This test (and other similar ones) should be replaced by adding `ThermalNoiseModel.json` and `ThermalNoiseModel.repr` files to `json_test_data`.  Please check instructions in https://github.com/quantumlib/Cirq/blob/main/docs/dev/serialization.md.

Reply:
Done as of 18f2565203a20ee800888826154e4572d474cccc